### PR TITLE
Create-getter, invert-object, trim-symbols, uniq tasks solution

### PR DESCRIPTION
**Pull request by baidauletov**
### Create-getter
String function: split, and clouser used to create getter to object.
### Invert-object
To swap object keys and values, Object functions: entries, fromEntrires; Array function: map, used.
### Trim-symbols
To remove consecutive identical symbols from string, that more than given size to function, String function: split; Array function reduce, used.
### Uniq
To return array with uniq values, new Set collection used.

### CODE REVIEW

**Comment by jsru** [comment]
Добавляю преподавателя (@dosandk) для код-ревью.

File: 03-objects-arrays-intro-to-testing/1-create-getter/index.js [hunk 1]
```diff
@@ -4,5 +4,19 @@
  * @returns {function} - function-getter which allow get value from object by set path
  */
 export function createGetter(path) {
+  const props = path.split(".");
```
Comment (dosandk) [inline 03-objects-arrays-intro-to-testing/1-create-getter/index.js:7]: 👍 лайк за использование замыкания

File: 03-objects-arrays-intro-to-testing/1-create-getter/index.js [hunk 2]
```diff
@@ -4,5 +4,19 @@
...
 
+  return (obj) => {
+    let result = obj;
+
+    for (const prop of props) {
+      if (result === undefined) {
+        return undefined;
```
Comment (dosandk) [inline 03-objects-arrays-intro-to-testing/1-create-getter/index.js:14]: просто `return` без `undefined`

File: 03-objects-arrays-intro-to-testing/3-trim-symbols/index.js [hunk 3]
```diff
@@ -5,5 +5,28 @@
  * @returns {string} - the new string without extra symbols according passed size
  */
 export function trimSymbols(string, size) {
```
Comment (dosandk) [inline 03-objects-arrays-intro-to-testing/3-trim-symbols/index.js:7]: good 👍

File: 03-objects-arrays-intro-to-testing/4-uniq/index.js [hunk 4]
```diff
@@ -4,5 +4,5 @@
  * @returns {*[]} - the new array with uniq values
  */
 export function uniq(arr) {
-
+  return [...new Set(arr)];
```
Comment (dosandk) [inline 03-objects-arrays-intro-to-testing/4-uniq/index.js:7]: good 👍

**Review by dosandk — APPROVED** [review]
good 👍

**Pull request merged**